metric-example/metric-expressions-for-k8s.md

```diff
@@ -121,8 +128,9 @@ Using the following query, you can find how many pods are not running compared t
 Using the following query, you can find how many containers are not running compared to the number of desired containers of this workload.
 ```
 ( 
-  (builtin:cloud.kubernetes.pod.desiredContainers):parents:splitBy("dt.entity.cloud_application")
-  -(builtin:cloud.kubernetes.pod.containers):parents:splitBy("dt.entity.cloud_application")
+  builtin:cloud.kubernetes.pod.desiredContainers:parents:splitBy("dt.entity.cloud_application")
```

Comment:
Here I think it is fine to have two operators in a single line. I would consider adding some white-space around the colons for better readability, though:
```
builtin:cloud.kubernetes.pod.desiredContainers : parents : splitBy("dt.entity.cloud_application")
```

Reply:
We could apply analogous changes to other short-ish metric IDs and split others into lines.

Reply:
deleted the changes in metric-expressions-for-k8s.md - see comment by Florian